script.js

```diff
@@ -0,0 +1,75 @@
+var evenOdd;
+var myFuncCalls = 0;
+if (myFuncCalls%2===0){
+  evenOdd="X"
+  }
+else{
+  evenOdd="O"
+  }
+
+
+function changeText1(){
+  myFuncCalls++;
+  var obj= document.getElementById("i1");
+  obj.innerHTML= evenOdd;
+}
+function changeText2(){
+  myFuncCalls++;
+  var obj= document.getElementById("i2");
+  obj.innerHTML= evenOdd;
+}
+function changeText3(){
+  myFuncCalls++;
+  var obj= document.getElementById("i3");
+  obj.innerHTML= evenOdd;
+}
+function changeText4(){
+  myFuncCalls++;
+  var obj= document.getElementById("i4");
+  obj.innerHTML= evenOdd;
+}
+function changeText5(){
+  myFuncCalls++;
+  var obj= document.getElementById("i5");
+  obj.innerHTML= evenOdd;
+}
+function changeText6(){
+  myFuncCalls++;
+  var obj= document.getElementById("i6");
+  obj.innerHTML= evenOdd;
+}
+function changeText7(){
+  myFuncCalls++;
+  var obj= document.getElementById("i7");
+  obj.innerHTML= evenOdd;
+}
+function changeText8(){
+  myFuncCalls++;
+  var obj= document.getElementById("i8");
+  obj.innerHTML= evenOdd;
+}
+function changeText9(){
+  myFuncCalls++;
+  var obj= document.getElementById("i9");
+  obj.innerHTML= evenOdd;
+}
+function clearBoard(){
+  var clear1= document.getElementById("i1");
```

Comment:
Could you use a classname to find an array of all elements in one line, and then iterate over that array and clear them that way?